src/server.c

```diff
@@ -4526,7 +4526,7 @@ int finishShutdown(void) {
 
 
     serverLog(LL_WARNING,"%s is now ready to exit, bye bye...",
-        server.sentinel_mode ? "Sentinel" : "Redis");
+        server.sentinel_mode ? "Sentinel" : "Valkey");
```

Comment:
can we leave the string change out? 

I like @zuiderkwast's suggestion to not use hard-coded string at all but go with a symbolic name, which can get changed easily. Since we haven't closed on this decision and there are already several other pending renaming PRs, I would suggest to keep the scope of this change just to the logo. 

Also I would caution that we don't have an official logo yet so chances are we might have to change the logo again when the official one comes out. However, it is good/important to remove the Redis logo asap and this one looks great to me, personally.

Reply:
Yes, we can track the renaming in the other PR, Will revert the change